static/js/containers/FinancialAidCalculator.js

```diff
@@ -0,0 +1,208 @@
+import React from 'react';
```

Comment:
the problem is the `SelectField` which has type annotations which explicitly expect profile objects.